probe/main.go

```diff
@@ -79,7 +79,8 @@ func main() {
 
 	if *dockerMapper {
 		docker := newDockerMapper(*procRoot, *dockerInterval)
-		pms = append(pms, &dockerIDMapper{docker}, &dockerNameMapper{docker})
+		pms = append(pms, docker.idMapper(), docker.nameMapper(),
+			docker.imageIDMapper(), docker.imageNameMapper())
```

Comment:
If you break lines for parameter lists, it's nice to leverage Go's tolerance of trailing commas to produce nicer-looking blocks:

```go
pms = append(pms,
    docker.idMapper(),
    docker.nameMapper(),
    docker.imageIDMapper(),
    docker.imageNameMapper(),
)
```